libraries/exoplayer/src/test/java/androidx/media3/exoplayer/analytics/DefaultAnalyticsCollectorTest.java

```diff
@@ -2199,6 +2272,15 @@ public void onLoadStarted(
       reportedEvents.add(new ReportedEvent(EVENT_LOAD_STARTED, eventTime));
     }
 
+    @Override
+    public void onLoadStarted(
+        EventTime eventTime,
+        LoadEventInfo loadEventInfo,
+        MediaLoadData mediaLoadData,
+        int retryCount) {
+      reportedEvents.add(new ReportedEvent(DEPRECATED_EVENT_LOAD_STARTED, eventTime));
```

Comment:
I think you've got this the wrong way round? This is an override of the new, un-deprecated method with the `retryCount` parameter but it's using the `DEPRECATED_` constant.